task/ko/0.1/README.md

```diff
@@ -0,0 +1,41 @@
+# ko
+
+This Task builds source into a container image using [ko](https://ko.build/).
+
+
+## Install the Task
+
+```
+kubectl apply -f https://api.hub.tekton.dev/v1/resource/tekton/task/ko/0.1/raw
+```
+
+## Parameters
+
+* **ko-image**: The ko image to use for builds (_default:_ `ghcr.io/ko-build/ko:latest`)
```

Comment:
What do you think about using the chainguard image for ko? PTAL @imjasonh 
> https://github.com/chainguard-images/images/blob/main/images/ko/README.md

Reply:
This is the image built by the ko-build/ko repo

Reply:
Yes, I know, but if you compare them against the CVE count, you will notice that the chainguard's image is well-managed and contains fewer packages and CVEs. Also, it shipped with SBOM and provenance.

Reply:
I made `ko-image` a parameter so users with opinions can override (like @developer-guy).  However, I think the general user of the ko task will want to use upstream.  

From the chainguard readme:
> This image is designed for use in situations where you would like to use ko with codebases that have C dependencies where cgo must be used. In these cases, staticly linking against musl instead of glibc results in smaller binaries.

Reply:
Yeah, I think I'd recommend using `ghcr.io/ko-build/ko` as the default, since `cgr.dev/chainguard/ko` (which I also help maintain) has a bit more focused use case than the regular upstream one.